dash/extract-meta.js

```diff
@@ -738,12 +760,14 @@ function gatherComponents(sources, components = {}) {
                     .join('');
             }
             const doc = {
-                displayName: name,
+                displayName: `${name} - ${rootExp.name}`,
```

Comment:
Added the export name to the display name - Reasoning for this is for better debugging handling by end user. Especially if they exporting multiple things here. It was quiet hard to know what was failing before. Plus  I believe it will be needed to support multiple export in same file